backend/seeder/seeder.py

```diff
@@ -0,0 +1,228 @@
+            title=fake.catch_phrase(),
+            description=fake.catch_phrase(),
+            deadlines=deadlines,
+            course_id=course_id,
+            visible_for_students=random.choice([True, False]),
+            archived=random.choice([True, False]),
+            regex_expressions=[]
+        )
+        projects.append(project)
+    return projects
+
+
+def generate_submissions(project_id, student_uid):
+    """Generates a list of submissions with random status"""
+    submissions = []
+    statusses = [SubmissionStatus.SUCCESS, SubmissionStatus.FAIL,
+                 SubmissionStatus.LATE, SubmissionStatus.RUNNING]
+    num_submissions = random.randint(0, 2)
+    for _ in range(num_submissions):
+        submission = Submission(project_id=project_id,
+                                uid=student_uid,
+                                submission_time=datetime.now(),
+                                submission_path="",
+                                submission_status=random.choice(statusses))
+        graded = random.choice([True, False])
+        if graded and submission.submission_status == "SUCCESS":
+            submission.grading = random.randint(0, 20)
+        submissions.append(submission)
+    return submissions
+
+
+def into_the_db(my_uid):
+    """Populates the db with 5 courses where my_uid is teacher and 5 where he is student"""
+    try:
+        session = session_maker()  # setup the db session
+        connection = session.connection()
+        register_composites(connection)
+
+        students = []
+        # make a random amount of 100-200 students which we can use later to populate courses
+        num_students = random.randint(100, 200)
+        students = [student_generator() for _ in range(num_students)]
+        session.add_all(students)
+        session.commit()
+
+        num_teachers = random.randint(5, 10)
+        teachers = [teacher_generator() for _ in range(num_teachers)]
+        session.add_all(teachers)
+        session.commit()  # only after commit uid becomes available
+
+        for _ in range(5):  # 5 courses where my_uid is teacher
+            course_id = insert_course_into_db_get_id(session, my_uid)
+            # Add students to the course
+            subscribed_students = populate_course_students(
+                session, course_id, students)
+            populate_course_projects(
+                session, course_id, subscribed_students, my_uid)
+
+        for _ in range(5):  # 5 courses where my_uid is a student
+            teacher_uid = teachers[random.randint(0, len(teachers)-1)].uid
+            course_id = insert_course_into_db_get_id(session, teacher_uid)
+            subscribed_students = populate_course_students(
+                session, course_id, students)
+            subscribed_students.append(my_uid)  # my_uid is also a student
+            populate_course_projects(
+                session, course_id, subscribed_students, teacher_uid)
+    except Exception as e:
```

Comment:
Strange that the linter doesn't catch this, but this should be SQLAlchemyError, not a generic exception